source/develop/release-management/features/network/unrestricted-network-names.html.md

```diff
@@ -43,18 +47,17 @@ We will make no change in the Engine/Vdsm API. This makes backward compatibility
 
 A new field, "vdsm_name", is to be added to the network entity. When a new network is added, its existing "name" has to be provided by the human actor. The new "vdsm_name" is to be auto-generated by Engine according to a pattern
 
-      brXXXXXXXXXXXXX
+      onXXXXXXXXXXXXX
 
-where `XXXXXXXXXXXXX` are 13 hexadecimal characters extracted from the UUID of the network. "name" is to be editable by the end user, and exposed as the name of the network over REST (as it is today); "vdsm_name" would be immutable, and to be used only for Engine/Vdsm communication.
+where `XXXXXXXXXXXXX` are 13 hexadecimal characters extracted from the UUID of the network. "name" is to be editable by the end user, and exposed as the name of the network over REST (as it is today); "vdsm_name" would be immutable once the network is attached to a host and to be used only for Engine/Vdsm communication.
+
+In a scenario where "name" doesn't exceed IFNAMSIZ and only contains ASCII characters, in order to ease debugging and backwards compatibility, "vdsm_name" will match "name". In order to avoid any lingering states that
+may be resulted in network name changes, we'll continue disallowing changing network names of networks that are host attached.
```

Comment:
Suggested change:
"may result in network name changes, we will continue to disallow the changing of network names for networks that are host attached."